exist-core/src/main/java/org/exist/storage/blob/BlobStoreDumpTool.java

```diff
@@ -127,19 +128,19 @@ public static void dump(final DigestType digestType, final Path persistentFile,
             // write a CSV table header
             printStream.println("blobId,referenceCount");
 
-            buffer.clear();
+            ((Buffer)buffer).clear();
 
             final byte[] id = new byte[digestBytesLen];
 
             while (channel.read(buffer) > -1) {
-                buffer.flip();
+                ((Buffer)buffer).flip();
```

Comment:
is this still needed?

Reply:
I guess if you remove the java8 compatibility thing in parent pom, all should be ok?

Reply:
Its related to the `<complianceLevel>8<complianceLevel>` of the ajc compile

Reply:
ah ok, thnx. and that is still needed?